content/news/march-2016-tournament-announcement.md

```diff
@@ -0,0 +1,13 @@
+It is time for yet another OpenRA tournament, which, for the first time ever, will feature a prize money of $200!
+
+Once again our community member Ripley calls onto all OpenRA players to compete for the crown of ultimate RTS champions!
+
+Running on the weekends of March 5th-6th and March 11th-12th 2016, the tourney will take the format the of tag team elimination. Teams of two players will square off in a total of three games, two of which being 1v1 matches between pairs of opposing players and a final 2v2 match between both teams.
+
+Thanks to our casters of the Comcast Crew and [GameReplays.org](http://www.gamereplays.org/portals.php) a prize money of $200 will be available for this tournament! The prize will be split 80/20 between the first and second place, to be evenly shared among team members.
+
+For official tournament rules and team registration, [visit the forum thread](http://forum.openra.net/82/19414). If you are looking for potential team mates, have a look at [this thread](http://forum.openra.net/82/19415) to find someone to team up with. Registration ends on March 2nd at 0:00 UTC, so don't waste any time and register right now!
```

Comment:
`If you are looking for potential team mates, have a look at [this thread](http://forum.openra.net/82/19415) to find someone to team up with.` ->  `If you are looking for a team mate, have a look at [this thread](http://forum.openra.net/82/19415).`